lib/matobj2.gd

```diff
@@ -120,11 +120,45 @@
 
 
 # The following are guaranteed to be always set or cheaply calculable:
```

Comment:
Remove that line? Or at least put an empty line between it and the new comment below.